cmd/root.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
+	Use:   "spring-boot-project-starter",
+	Short: "Spring-boot-project-starter (sc) er et verktøy for å...",
```

Comment:
Spring-boot-project-starter (sc) er et verktøy for å..
->
Spring-boot-co-pilot er et støtteverktøy i utvikling der kjente oppgaver er automatisert

Reply:
Foreslår rename av prosjekt til spring-boot-co-pilot